db/seeds/utils.js

```diff
@@ -1,22 +1,31 @@
 exports.convertTimestampToDate = ({ created_at, ...otherProperties }) => {
-	if (!created_at) return { ...otherProperties };
-	return { created_at: new Date(created_at), ...otherProperties };
+  if (!created_at) return { ...otherProperties };
+  return { created_at: new Date(created_at), ...otherProperties };
 };
 
 exports.createRef = (arr, key, value) => {
-	return arr.reduce((ref, element) => {
-		ref[element[key]] = element[value];
-		return ref;
-	}, {});
+  return arr.reduce((ref, element) => {
+    ref[element[key]] = element[value];
+    return ref;
+  }, {});
 };
 
 exports.formatComments = (comments, idLookup) => {
-	return comments.map(({ created_by, belongs_to, ...restOfComment }) => {
-		const article_id = idLookup[belongs_to];
-		return {
-			article_id,
-			author: created_by,
-			...this.convertTimestampToDate(restOfComment),
-		};
-	});
+  return comments.map(({ created_by, belongs_to, ...restOfComment }) => {
+    const article_id = idLookup[belongs_to];
+    return {
+      article_id,
+      author: created_by,
+      ...this.convertTimestampToDate(restOfComment),
+    };
+  });
+};
+
+exports.checkDescendingOrder = (array) => {
```

Comment:
Great going with this util function.
If you need it further down the line don't forget you have jest-sorted!